.ladle/adapters/PlainComponentAdapter.tsx

```diff
@@ -16,7 +16,9 @@ import type {
 } from '../../src/components/Common/UI/Button/ButtonTypes'
 import type { LinkProps } from '../../src/components/Common/UI/Link/LinkTypes'
 import type { ComponentsContextType } from '@/contexts/ComponentAdapter/useComponentContext'
+import type { MenuProps } from '@/components/Common/UI/Menu/MenuTypes'
 
+// Instead of using require, define a type-safe adapter for Material UI menu
```

Comment:
AI cruft :-P. I actually ask AI to remove bad comments now as a last step. Seems like it thinks this is material UI.